.github/stale.yml

```diff
@@ -0,0 +1,41 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 7
+
+# Number of days of inactivity before a stale Issue or Pull Request is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
+
+# Label to use when marking as stale
+staleLabel: stale
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  This pull request has been automatically marked as stale because it has not had
+  activity in the last 7 days. It will be closed if no further activity occurs. Please
+  feel free to give a status update now, ping for review, or re-open when it's ready.
+  Thank you for your contributions!
+
+# Comment to post when removing the stale label.
+# unmarkComment: >
+#   Your comment here.
+
+# Comment to post when closing a stale Issue or Pull Request.
+#closeComment: >
+#   Your comment here.
+
+# Limit the number of actions per hour, from 1-30. Default is 30
+limitPerRun: 1
+
+# Optionally, specify configuration settings that are specific to just 'issues' or 'pulls':
+issues:
+  daysUntilStale: 30
+  markComment: >
+    This issue has been automatically marked as stale because it has not had activity in the
+    last 30 days. It will be closed unless it is tagged "help wanted" or other activity
+    occurs. Thank you for your contributions.
+
+ issues:
+   exemptLabels:
+     - help wanted
```

Comment:
ignore `tech debt` as well?

Reply:
@alyssawilk how does this compare to the config you started a while back? I've been trying to keep the issue clean, but we likely need to go through and look for things that do not have help wanted (I think tech debt issues should be marked help wanted). Also, will those close assigned issues with no activity? I think we don't want that.

Reply:
This is pretty close to the work Alyssa started in a prior PR. I trimmed some commented lines, but it's almost a 1-1 mapping.

Reply:
I would probably remove "tech debt" per my above comment since it should have help wanted or be assigned. Will this close assigned issues that aren't updated?

Reply:
I'd limited to PRs as well, at least while we got a handle on things.  I'd like to auto-close issues where possible though. We could just put a longer warning on issues; 30 or 60 days?  If there's been no progress in 60 days whoever is working on a larger task can just reopen when they've made progress.

Reply:
Yeah, or I guess just add help-wanted and unassign themselves. I mostly just don't want to lose old issues that we actually would like to implement eventually. But, I agree we should close things so we can give it a shot.

Reply:
list of not-yet-flagged-as-help-wanted issues here
https://github.com/envoyproxy/envoy/issues?page=3&q=is%3Aissue+is%3Aopen+-label%3A%22help+wanted%22&utf8=%E2%9C%93  for reference

Reply:
Should I just drop the `tech debt` label for now then? Or do we also want to lengthen the time to close?

The code seems to only close if the stale label was already applied

[close sweep](https://github.com/probot/stale/blob/master/lib/stale.js#L43-L48)
[closable filter](https://github.com/probot/stale/blob/master/lib/stale.js#L71-L77)

Reply:
Should we also consider a pinned label for issues that are definite future enhancements but are not help wanted? I'm thinking of something like QUIC or other ongoing efforts based on the list Alyssa linked.

Reply:
I mildly prefer just extending the date to something we're happy with.  If you're picking up a bug which is going to take months to complete it'd still be good to provide monthly updates if you're working on it.